plugins/action_increment_stage/src/incrementStage.ts

```diff
@@ -24,7 +24,7 @@ async function incrementStage({
         : await DBConnect.getCurrentStageStatusHistory(applicationId)
```

Comment:
Should this also cover the case when is comming from `applicationData`?

Reply:
Not sure what you mean. The logic for getting the applicationId is handled at the start with: 
`const applicationId = parameters?.applicationId ?? applicationData?.applicationId`

By the time we get to here it shouldn't make any difference where it came from -- parameters or applicationData.

Is that what you meant?